dao/src/main/java/org/thingsboard/server/dao/attributes/CachedAttributesService.java

```diff
@@ -170,6 +172,7 @@ public ListenableFuture<List<AttributeKvEntry>> find(TenantId tenantId, EntityId
                 return mergedAttributes;
             } catch (Throwable e) {
                 cacheTransaction.rollback();
+                log.error("Could not find attributes from cache", e);
```

Comment:
This log is not usable. Please log the entityId, scope and key and use WARN level.